libs/ui/layout/src/lib/chart/chart.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing'
```

Comment:
I don't think chart is a `layout` component.
Also, we should really focus on bundle size and dependency tree for `affected:test` duration.

IMO, we should start using standalone component, at the very first.
We must really improve the dependency tree of Angular modules.
Note that almost every app or lib will import layout lib, so it means `chart.js` will be in all bundles, even the ones not using it.

Reply:
hmm that's a good point, I was the one suggesting to put it in layout but it's true that it's not a good idea in terms of bundle. Maybe a new dedicated module ui/chart or something like that would make sense?

Reply:
It seems that we are going to get stuck anyway with the `lib` system cause we might need to create way too many libs to keep it consistent.

There are 2 things to keep in mind:
- bundle size for apps depends on the `import` files, which mostly match angular modules dependencies. One lib is a module so we include it or not. This can be addressed using standalone component, it's a must go imo.
- affected targets of Nx only works on lib dependency so far I guess , so here we could gain by creating new libs for more dedicated usecases.

With better module architecture and standalone components, we could event imagine to have both smart/ui logic in the same lib, which would make sense somehow.